src/simulate_ir.jl

```diff
@@ -12,15 +12,19 @@ export net, Named
 Base.@kwdef struct SimSpec
     time::Float64=0.0
     ϵω::Float64=0.0
+    # rng currently breaks incidence analysis
+    rng::Union{#=Random.Xoshiro,=# Nothing}=nothing
```

Comment:
Do we have an MWE for this?

Reply:
I think the MWE is this: https://github.com/CedarEDA/CedarSim.jl/blob/5d5ea8d4e7e17fd06f775eddd11f15a4731a4210/test/basic.jl#L51